src/psadropdown/PSADropdown.jsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  FormControl, InputLabel, MenuItem, Select,
+} from '@mui/material';
+
+export const PSADropdown = ({
+  id,
+  labelId,
+  label,
+  value,
+  onChange,
+  variant,
+  items = [],
+  style = {},
+  formSx = {},
+  inputSx = {},
+  selectSx = {},
+  MenuProps = {},
+  error = false,
```

Comment:
Can you add a props called `SelectProps` and apply it to the `<Select/>` ? In this way you can get rid of a few of these props and also have better reusability